src/Zadatak2/Rectangle.java

```diff
@@ -0,0 +1,46 @@
+package Zadatak2;
+
+public class Rectangle {
+    //attributes
```

Comment:
These comments are not required. Comments are used to explain complicated or not obvious logic used for maybe performance reason. 
Comments also need to be maintained and should be used sparingly.

Ideally code should be self documenting (method names, attribute names...)

See this:
https://www.reddit.com/r/ProgrammerHumor/comments/8w54mx/code_comments_be_like/

Reply:
I get it haha the floor is made out of floor xD